python/spacewalk/server/importlib/backend.py

```diff
@@ -171,11 +171,9 @@ def processCapabilities(self, capabilityHash):
 
     # pylint: disable-next=invalid-name
     def processChangeLog(self, changelogHash):
-        if (
-            CFG.has_key("package_import_skip_changelog")
-            and CFG.package_import_skip_changelog
-        ):
-            return
+        with cfg_component(None) as cfg:
```

Comment:
@agraul last thing you commented was about checking if `component=None` was the right component when accessing `package_import_skip_changelog` (if I understood you correctly). I believe this property is not in the configuration defaults  (and I believe not documented either).

Reply:
Either way, it does seem to be used at the top level of `/etc/rhn/rhn.conf`, e.g. in BZ#1129748. So IMO `component=None` sounds correct to me.